### PR TITLE
Allow up to 5 Clients to communicate in parallel with Service

### PR DESCRIPTION
**Pull request by spaceisfun**

### CODE REVIEW

File: src/Service/ServiceExe.cs [hunk 1]
```diff
@@ -1514,13 +1514,13 @@ public void Start(bool forceUserTaskRerun)
             // Start IPC server on port 45684. Only start after all boot tasks are complete.
             if (EnableNetworkAccess && !DisableNetworkAccess) 
             {
-                FOServiceExe.ipcHost = new IpcServiceHostBuilder(FOServiceExe.ipcSvcProvider).AddTcpEndpoint<IFactoryOrchestratorService>("tcp", IPAddress.Any, 45684)
+                FOServiceExe.ipcHost = new IpcServiceHostBuilder(FOServiceExe.ipcSvcProvider).AddTcpEndpoint<IFactoryOrchestratorService>("tcp", IPAddress.Any, 45684, new JKang.IpcServiceFramework.Tcp.TcpConcurrencyOptions(5))
```
Comment (hajya) [inline src/Service/ServiceExe.cs:1517]: what's the significance of 5 here? should we track possibly offering this as a customization?

Comment (hajya) [inline src/Service/ServiceExe.cs:1517]: if a 6th client tries, does it just get put behind a retry?

Comment (spaceisfun) [reply src/Service/ServiceExe.cs:1517]: The API they call just blocks until a slot is available. No timeout or rerty needed.

Comment (spaceisfun) [reply src/Service/ServiceExe.cs:1517]: It uses a semaphone lock

Comment (spaceisfun) [reply src/Service/ServiceExe.cs:1517]: Seemed high enough to cover any reasonable use case of FO, without being silly large. I'm worried a bit about customization overload already so i'm hesitant to make it one.

Comment (hajya) [reply src/Service/ServiceExe.cs:1517]: I would consider documenting this as the current supported # of parallel connections.

**Pull request merged**